cypress/integration/pages/photoGalleryPage/tests.js

```diff
@@ -0,0 +1,63 @@
+import config from '../../../support/config/services';
+import appConfig from '../../../../src/server/utilities/serviceConfigs';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    it('should render a H1, which displays the headline', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          cy.get('h1').should('contain', body.promo.headlines.headline);
+        },
+      );
+    });
+
+    it('should render a timestamp', () => {
+      cy.request(
+        `${config[service].pageTypes.photoGalleryPage.path}.json`,
+      ).then(({ body }) => {
+        const { lastPublished, firstPublished } = body.metadata;
+        const timeDifferenceMinutes =
+          (lastPublished - firstPublished) / 1000 / 60;
+        const minutesTolerance = 1;
+        cy.get('time')
+          .eq(0)
+          .should('exist')
+          .should('be.visible')
+          .should('have.attr', 'datetime')
+          .should('not.be.empty');
+
+        if (timeDifferenceMinutes > minutesTolerance) {
```

Comment:
What is the significance of the single minute tolerance? Is this because some test fixtue data is unsuitable?

Reply:
Yep, there’s a bug with some CPS assets where `firstPublished` and `lastPublished` are milliseconds apart when they've only been published once. So the media team created logic to give tolerance for this discrepancy. They wrote a test for MAP pages which took this into account so I re-used it for PGL assets as we're using the same logic.

Reply:
You may want to add a comment in both the MAP and PGL tests just to make that clear, seems pretty arbitrary without the context.